src/hotspot/share/jfr/utilities/jfrAllocation.hpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2014, 2020, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2014, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
This appears to be the only change to this file so should be reverted.

Reply:
Thanks for catching this. 
Done.